st2api/st2api/controllers/exp/root.py

```diff
@@ -23,3 +24,6 @@ class RootController(object):
     # Deprecated. Use /v1/ instead.
     actionalias = ActionAliasController()
     aliasexecution = ActionAliasExecutionController()
+
+    # Experimental
+    validators = ValidationController()
```

Comment:
I thought we mostly use plural for API endpoints - /executions, /actions, /packs, etc. (models are sadly a different story).